clang/test/CodeGenSYCL/integration_header.cpp

```diff
@@ -160,4 +176,4 @@ int main() {
   });
 
   return 0;
-}
+}
```

Comment:
Format tool probably complained about this, but the newline after this needs to come back.